src/attr/_make.py

```diff
@@ -998,6 +1048,13 @@ def attrs(
           default value are additionally available as a tuple in the ``args``
           attribute,
         - the value of *str* is ignored leaving ``__str__`` to base classes.
+    :param bool collect_by_mro: Setting this to `True` fixes the way ``attrs``
```

Comment:
```suggestion
    :param bool collect_by_mro: Setting this to ``True`` fixes the way ``attrs``
```

Reply:
This one is actually correct thanks to #571!

Reply:
TIL

Reply:
Yes, it's very cool!  I also only learned about it when Julian submitted the PR…